kiva/examples/kiva/qt_simple.py

```diff
@@ -0,0 +1,76 @@
+# (C) Copyright 2005-2022 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+import sys
+
+from numpy import array
+
+# This is PyQt specific so force the toolkit selection.
+from traits.etsconfig.api import ETSConfig
+
+ETSConfig.toolkit = "qt.qpainter"
+
+from enable.examples._example_canvas import Canvas
+
+try:
+    from pyface.qt import QtGui
+except ImportError:
+    raise Exception("PyQt4 needs to be installed to run this example")
```

Comment:
Is this accurate? We stopped supporting PyQt4 some time ago, so I'm not sure there's much value in introducing a new example that requires PyQt4.

Reply:
Ah, I see; this isn't a new example - it's a duplicate of `kiva/examples/kiva/qpainter_simple.py`, with only this one line changed. I suggest removing this addition to the PR.